cmd/lncli/commands.go

```diff
@@ -525,7 +532,31 @@ func sendPaymentCommand(ctx *cli.Context) error {
 var ShowRoutingTableCommand = cli.Command{
 	Name:        "showroutingtable",
 	Description: "shows routing table for a node",
-	Action:      showRoutingTable,
+	Usage:       "showroutingtable [--table]",
+	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name:  "table",
+			Usage: "Show the routing table in table format. Print only a few first symbols of id",
+		},
+		cli.BoolFlag{
+			Name:  "human",
+			Usage: "Simplify output to human readable form. Output lightning_id partially. Only work with --table option.",
+		},
+		cli.StringFlag{
+			Name:  "type",
+			Usage: "Type of image file. Use one of: http://www.graphviz.org/content/output-formats. Usage of this option supresses textual output",
+		},
+		cli.StringFlag{
+			Name:  "viz",
+			Usage: "Specifies where to save the generated file. If don't specified use os.TempDir Usage of this option supresses textual output",
+		},
+		cli.BoolFlag{
+			Name:  "open",
+			Usage: "Open generated file automatically. Uses command line \"open\" command",
```

Comment:
It seems that cli.BoolFlag does not contain Description field.  Only cli.Command contains it.